src/openforms/variables/models.py

```diff
@@ -0,0 +1,101 @@
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+import jq
+from rest_framework.serializers import ValidationError as DRFValidationError
+
+from openforms.forms.api.validators import JsonLogicValidator
+
+from .constants import DataMappingTypes, ServiceFetchMethods
+from .validators import HeaderValidator
+
+
+class ServiceFetchConfiguration(models.Model):
+    service = models.ForeignKey(
+        "zgw_consumers.Service",
+        on_delete=models.PROTECT,
+    )
```

Comment:
please add `verbose_name=_("service")` so we can get that translated too in the admin